slides/background_python_and_jupyter_notebook.html

```diff
@@ -209,7 +209,7 @@
 
   - **Terminal** and a **text editor**
 
-      - We recommend [IPython](https://ipython.org/)
+      - We recommend [Jupyter notebook](https://jupyter.org/)
```

Comment:
Jupyter notebooks are not technically "terminal and text editor" and they are introduced below. I would like to leave this as it was